web-modeler-openapi-cli/src/main/java/org/camunda/community/webmodeler/cli/ListProjectsCommand.java

```diff
@@ -2,16 +2,27 @@
 
 import java.util.List;
 import org.camunda.community.webmodeler.client.dto.ProjectMetadataDto;
+import org.camunda.community.webmodeler.client.dto.PubSearchDtoProjectMetadataDto;
 import picocli.CommandLine;
 
 @CommandLine.Command(
         name = "listProjects",
         description = "Prints out projects in the organization.",
         mixinStandardHelpOptions = true)
 public class ListProjectsCommand extends AbstractCommand {
+
+    @CommandLine.Option(
+            names = {"--page", "-pg"},
+            description = "page number of the results to fetch, defaults to 0",
+            scope = CommandLine.ScopeType.LOCAL)
+    private int page;
+
     @Override
     public Integer call() throws Exception {
-        List<ProjectMetadataDto> projects = parent.buildProjectsClient().listProjects();
+        PubSearchDtoProjectMetadataDto searchDto =
+                new PubSearchDtoProjectMetadataDto().size(50).page(page);
```

Comment:
Might be better to extract `50` as a constant somewhere as it seems shared and we are defaulting to that amount for `ListProjectsCommand`, `ListFilesCommand` and `Downloader`

Reply:
Yep, makes sense!